sql-cli/sql_cli/cli.py

```diff
@@ -47,3 +69,14 @@ def run_dag(project: Project, dag: DAG, verbose: bool) -> DagRun:
     except Exception as exception:
         rprint(f"  [bold red]{exception}[/bold red]")
         raise Exit(code=1)
+    rprint(f"Completed running the workflow {dr.dag_id}. 🚀")
+    final_state = None
+    if dr.state == State.SUCCESS:
+        final_state = "[bold green]SUCCESS[/bold green]"
+    elif final_state == State.FAILED:
+        final_state = "[bold red]FAILED[/bold red]"
+    else:
+        final_state = dr.state
+    rprint(f"Final state: {final_state}")
+    elapsed_seconds = (dr.end_date - dr.start_date).microseconds / 10**6
+    rprint(f"Total elapsed time: [bold blue]{elapsed_seconds:.2}s[/bold blue]")
```

Comment:
Could you move this please to the `__main__` call. This way we ensure that the normal print statements only get printed once i.e. per command call.

FYI we moved the print in PR https://github.com/astronomer/astro-sdk/pull/1199